packages/python/plotly/plotly/graph_objs/_deprecations.py

```diff
@@ -7,25 +7,25 @@
 
 class Data(list):
     """
-    plotly.graph_objs.Data is deprecated.
-Please replace it with a list or tuple of instances of the following types
-  - plotly.graph_objs.Scatter
-  - plotly.graph_objs.Bar
-  - plotly.graph_objs.Area
-  - plotly.graph_objs.Histogram
-  - etc.
+        plotly.graph_objs.Data is deprecated.
```

Comment:
please revert these whitespace-only changes ... I think your editor must be doing this?

Reply:
This is weird because I never opened this file, so maybe `black` did it. I'll just check out the master version.